packages/core/src/animation/AnimatorStateTransitionCollection.ts

```diff
@@ -94,4 +98,9 @@ export class AnimatorStateTransitionCollection {
       transitions.splice(index + 1, 0, transition);
     }
   }
+
+  private _addNoExitTimeTransition(transition: AnimatorStateTransition): void {
```

Comment:
merge `_addNoExitTimeTransition` and `_addHasExitTimeTransition` is better? 

```ts
 if (hasExitTime) {
      this._addHasExitTimeTransition(transition);
    } else {
      this._addNoExitTimeTransition(transition);
    }
```
Appeared twice！